java/sage/BigBrother.java

```diff
@@ -224,7 +224,7 @@ static boolean setWatched(Airing a, long airWatchStart, long airWatchEnd,
         Wizard.getInstance().addWatched(a, (a.isDVD() || a.isBluRay()) ? 0 : a.getStartTime(),
             (a.isDVD() || a.isBluRay()) ? a.getDuration() : a.getEndTime(),
                 0, 0);
-        if (a.isTV() && (!Sage.EMBEDDED || Carny.getInstance().isLoveAir(a)))
+        if (a.isTV() && Carny.getInstance().isLoveAir(a))
```

Comment:
I just wanted to speak up because I noticed this. I think if we're seeing `(!Sage.EMBEDDED || "other condition")`, both conditions can be removed. The reason I'm saying this is because the `!` makes the first condition always true in the current code and the other condition doesn't ever get evaluated.